lib/class-sp-admin.php

```diff
@@ -203,18 +221,38 @@ public function settings_page() {
 		<?php
 	}
 
+	/**
+	 * Given the active tab slug and a tab slug to compare, determines if the
+	 * comparison tab is the active tab, and prints a class if so.
+	 *
+	 * @param string      $active  The active tab slug.
+	 * @param string|bool $compare The comparison tab slug.
+	 */
 	protected function tab_active( $active, $compare = true ) {
 		if ( $active === $compare ) {
 			echo ' nav-tab-active';
 		}
 	}
 
+	/**
+	 * Gets a human-readable error type given a type code.
+	 *
+	 * @param string $type The type code to dereference.
+	 * @access protected
+	 * @return string The human-readable error type.
+	 */
 	protected function error_type( $type ) {
 		switch ( $type ) {
-			case 'error' : return __( 'Errors', 'searchpress' );
-			case 'warning' : return __( 'Warnings', 'searchpress' );
-			case 'line' : return __( 'Messages', 'searchpress' );
-			case 'success' : return __( 'Success', 'searchpress' );
+			case 'error': 
+				return __( 'Errors', 'searchpress' );
+			case 'warning': 
+				return __( 'Warnings', 'searchpress' );
+			case 'line': 
+				return __( 'Messages', 'searchpress' );
+			case 'success': 
+				return __( 'Success', 'searchpress' );
+			default:
+				return __( 'Unknown', 'searchpress' );
```

Comment:
Looks like this addition breaks a failing phpunit test. I don't have a preference between updating the test or removing this default case, do whichever you prefer.